src/services/ApiClient.js

```diff
@@ -34,7 +34,7 @@ export function fetchAllTrenches() {
   return axios({
     headers: { "Content-Type": "application/x-www-form-urlencoded" },
     method: "get",
-    url: `//${server}:9000/idig/${project}/trenches`,
```

Comment:
@AlZorba This mean the user must always add `:9000` in the input ?

Reply:
yes whenhis server got port 9000 but on public server we should be able to point to default.